data-generation/src/main/scala/thesis/SnapshotDelta.scala

```diff
@@ -266,4 +319,13 @@ object SnapshotDeltaObject {
     })
   }
 
+  def returnClosestGraph(instant: Instant)(g1: (G, Instant), g2: (G, Instant)): (G, Instant) =
+    if (Duration.between(g1._2, instant).abs() <= Duration.between(g2._2, instant).abs()) {
+      g1
+    } else {
+      g2
+    }
```

Comment:
```suggestion
  def returnClosestGraph(instant: Instant)(snapshot1: Snapshot, snapshot2: Snapshot): Snapshot =
    if (Duration.between(snapshot1.timestamp, instant).abs() <= Duration.between(snapshot2.timestamp, instant).abs()) {
      snapshot1
    } else {
      snapshot2
    }
```
Her f.eks. blir det litt lettere å se hva som foregår hvis vi får laget en Snapshot-klasse. Blir litt likt som SnapshotDeltaObject, men føler det går fint

Reply:
enig